docs/setup/setup.md

```diff
@@ -190,6 +190,19 @@ We have provided a sample node.json to add a node in kubernetes. Please make sur
 
     **Note:** Please run edge using the users who have root permission.
 
++ Run edge with systemd
+    It is also possible to start the edgecore with systemd. If you want, you could use the example systemd-unit-file. The following command will show you how to setup this:
```

Comment:
nit: please make the work systemd-unit-file a hyperlink to the edge.service file